napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -160,24 +162,37 @@ def __init__(self, layer):
             layer,
             'zoom',
             Mode.PAN_ZOOM,
-            tooltip=trans._('Pan/zoom mode (Space)'),
             checked=True,
         )
-        self.pick_button = QtModeRadioButton(
-            layer, 'picker', Mode.PICK, tooltip=trans._('Pick mode (L)')
+        action_manager.bind_button(
+            'napari:activate_picker_mode', self.panzoom_button
+        )
+
+        self.pick_button = QtModeRadioButton(layer, 'picker', Mode.PICK)
+        action_manager.bind_button(
+            'napari:activate_picker_mode', self.pick_button
```

Comment:
In general I would prefer we had all these strings defined in enums, as it would make it easier to change eventually (if it made sense) and also avoid typing mistakes?

```python
class NapariLayerActions:
     ActivatePickerMode = 'napari:activate_picker_mode
     ActivatePaintMode = 'napari:activate_paint_mode'

```

Reply:
I'm not sure this is worth it; 1) the registration is based on the function name, and the decorator automatically prepend `"napari:"`, so you would still need to make sure the values in the enum are the same as the function name 2) you won't be able to use the enum in the settings anyway. 3) That would defeat the purpose of bing able to define/bind in any order as now you need both the defining location and binding location to be able to import the enum.